src/api/schema/events/mod.rs

```diff
@@ -0,0 +1,32 @@
+mod log;
+
+use crate::event::EventInspector;
+use async_graphql::{Context, Subscription};
+use async_stream::stream;
+use log::LogEvent;
+use std::sync::Arc;
+use tokio::stream::Stream;
+
+#[derive(Default)]
+pub struct EventsSubscription;
+
+#[Subscription]
+impl EventsSubscription {
+    /// A stream of component log events
+    pub async fn log_events<'a>(
+        &'a self,
+        ctx: &'a Context<'a>,
+        component_name: String,
+    ) -> impl Stream<Item = LogEvent> + 'a {
```

Comment:
I can see this eventually being extracted out into a general filter, but think it's okay inline for now.